### ISSUE TITLE
SIGSEGV

### ISSUE THREAD

**Issue by bblay** [ribice/glice#13]
Hi, it just crashes.
```
$ glice
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x13c1f1d]

goroutine 1 [running]:
main.(*deps).exists(0xc0000d5e28, 0xc000270caf, 0x16, 0xa00, 0xc000270c80)
	/Users/bblay/go/src/github.com/ribice/glice/main.go:180 +0xdd
main.(*deps).getDeps(0xc0000d5e28, 0xc000182f3c, 0x18)
	/Users/bblay/go/src/github.com/ribice/glice/main.go:158 +0x37d
main.main()
	/Users/bblay/go/src/github.com/ribice/glice/main.go:81 +0x564
```

**Comment by ribice** [comment]
I removed the `license != nil` check as now I always instantiate it.

Is the repo you're using glice for publicly accessible? I'd like to test it.

**Comment by bblay** [comment]
No but I've made a pr to re-add that nil check, in case it's any help.

**Comment by bblay** [comment]
It doesn't crash now, but it does say `2019/03/26 14:29:52 fork/exec /bin/bash: resource temporarily unavailable`, which I guess is a separate issue.

**Comment by ribice** [comment]
Resolved by #14
